src/Worker.php

```diff
@@ -110,6 +113,15 @@ public function handle(Request $request, RequestContext $context): void
         } catch (Throwable $e) {
             $this->handleWorkerError($e, $sandbox, $request, $context, $responded);
         } finally {
+            $sandbox->forgetInstance('request');
```

Comment:
How about collecting these instances in `FlushTemporaryContainerInstances` listener?

Reply:
I've changed the code to flush everything.